featuretools/entityset/entityset.py

```diff
@@ -4,7 +4,6 @@
 from builtins import range, zip
 from collections import defaultdict
 
-import dask.dataframe as dd
```

Comment:
can we remove dask as a dependency in featuretools now?

Reply:
Looks like it. The only other use is in `load_flight` which gets used in the approximate features docs page (but with a limit on the number of rows). We could just put a hard cap on the number of rows and then not require `dask` projectwide?